engine/access/rest/websockets/models/block_models.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+)
+
+type BlockMessageResponse struct {
+	Block       *flow.Block      `json:"block"`
+	BlockStatus flow.BlockStatus `json:"block_status"`
```

Comment:
I don't think this is needed in the response since the client already passed it in the request.